lib/pixel/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+# zephyr-keep-sorted-start
+zephyr_library_sources(bayer.c)
+zephyr_library_sources(formats.c)
+zephyr_library_sources(kernel.c)
+zephyr_library_sources(print.c)
```

Comment:
shouldn't this only be included if the Kconfig is set?

Reply:
AFAIU it is already done from the [parent `CMakeLists.txt`]()https://github.com/zephyrproject-rtos/zephyr/pull/88839/files#diff-4803677bd280f4dded52c428e3c8ac6c618508dcabcbe880b68be85897c2ad41:
```
add_subdirectory_ifdef(CONFIG_PIXEL pixel)
```

Let me know if there should also be a `_ifdef` condition on the inside.

Reply:
`CONFIG_PIXEL_PRINT_NONE=y` allows to keep all the `pixel_print_` debug statements which help for troubleshooting the tests and useful in samples, but turn them off the rest of the time.
So when it is on, the `pixel.c` functions are still called but do nothing.
Bad idea?